mne/viz/utils.py

```diff
@@ -1158,14 +1158,33 @@ def _compute_scalings(scalings, inst, remove_dc=False, duration=10):
     """
     from ..io.base import BaseRaw
     from ..epochs import BaseEpochs
+
+    scalings = deepcopy(scalings)
     scalings = _handle_default('scalings_plot_raw', scalings)
     if not isinstance(inst, (BaseRaw, BaseEpochs)):
         raise ValueError('Must supply either Raw or Epochs')
 
+    for key, value in scalings.items():
+        if isinstance(value, str) and value != 'auto':
```

Comment:
This logic seems wrong now as the `not` was omitted, should probably be
```suggestion
        if not (isinstance(value, str) and value != 'auto'):
```

Reply:
... if you were trying to distribute the `not`, you probably meant:
```suggestion
        if not isinstance(value, str) or value != 'auto':
```

Reply:
Actually I believe we could probably change this this to

```python
        if value != 'auto' and not isinstance(value, float):
```

WDYT?

Reply:
Pushed a respective change in cbc14e8

Reply:
I like the old conditional better, for example:
```
>>> import numpy as np
>>> np.array([1.]).item()
1.0
>>> float(np.array([1.]))
1.0
>>> np.array([1.]) == 'auto'
<stdin>:1: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
False
```
That warning would not happen with the `isinstance(float)` call.

To me the logic of `if not (string and equal to 'auto'): then convert to float` is clearer and more general